extension/lib/time_tracker_view.js

```diff
@@ -103,16 +103,11 @@ class TimeTrackerView {
     Update display speed after speed is changed by user
     */
     updateSpeed(speed, sentence_id) {
-        // TODO: There should be an upper limit to this; because we have a speed bias, it cannot get infinitely fast. 
         this.reading_speed = speed // Completely made up eq, reasonable enough for testing though
```

Comment:
already handled in contentscript
See adjustSpeed

** best is if that check gets plumbed to updateSpeed but ehh.. this is good enough